autoload/multiple_cursors.vim

```diff
@@ -1083,8 +1086,8 @@ endfunction
 let s:retry_keys = ""
 function! s:display_error()
   if s:bad_input == s:cm.size()
-        \ && s:from_mode ==# 'n'
-        \ && has_key(g:multi_cursor_normal_maps, s:char[0])
+        \ && ((s:from_mode ==# 'n' && has_key(g:multi_cursor_normal_maps, s:char[0]))
+        \ ||  (s:from_mode ==# 'v' && has_key(g:multi_cursor_visual_maps, s:char[0])))
```

Comment:
Should this be checking for visual block mode as well? I think that's `V`, but I'm not sure if the same multi-key mappings should work in it?